textures/metro_control_panel_001_pegboard.py

```diff
@@ -32,3 +34,29 @@ def _swap_pegs(pegs: Pegs, a: str, b: str) -> Pegs:
     return tuple(
         tuple(s.replace(a, placeholder).replace(b, a).replace(placeholder, b) for s in panel) for panel in pegs
     )
+
+
+def _markers_to_bool_list(peg_str: str) -> Iterable[bool]:
```

Comment:
That name is wrong, it's not return a `list`.

Reply:
Good catch.  It probably doesn't make sense to have types in the name anyways since we are using type hints